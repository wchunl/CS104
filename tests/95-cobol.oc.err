h∏ç:3.8: syntax error, unexpected TOK_IDENT
h∏ç:3.31: invalid source character (.)
h∏ç:4.18: invalid source character (.)
h∏ç:4.30: invalid source character (.)
h∏ç:7.28: invalid source character (.)
h∏ç:8.29: invalid source character (.)
h∏ç:9.23: invalid source character (.)
h∏ç:9.33: invalid source character (.)
h∏ç:10.23: invalid source character (.)
h∏ç:10.33: invalid source character (.)
h∏ç:12.21: invalid source character (.)
h∏ç:13.20: invalid source character (.)
h∏ç:15.26: invalid source character (.)
h∏ç:17.26: invalid source character (.)
h∏ç:18.13: invalid source character (.)
h∏ç:19.47: invalid source character (.)
h∏ç:21.16: invalid source character (.)
h∏ç:22.23: invalid source character (.)
h∏ç:23.12: invalid source character (.)
