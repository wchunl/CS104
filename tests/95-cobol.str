string_set[   1]:    8476822520031942378 0x8dcd08->"EXIT"
                    4533532405753959164 0x8dc2f8->"001100"
string_set[   3]:    8848093992654105885 0x8dcc68->"RUN"
string_set[   7]:   11668471760152936011 0x8d7758->"IDENTIFICATION"
string_set[  11]:   14618159606949852052 0x8dba78->"000200"
string_set[  15]:    8749905388617569012 0x8dc618->"BEGIN"
string_set[  16]:    8528942095927433006 0x8dc0f8->""Hello world!""
string_set[  17]:    6142509188972423790 0x8d7738->""
string_set[  19]:    2366151243705444291 0x8dc848->"ERASE"
string_set[  20]:   14773745439228563802 0x8dccb8->"100700"
string_set[  22]:   10159970873491820195 0x8dc7a8->"1"
string_set[  24]:     848867923138271600 0x8dbbd8->"000300"
string_set[  25]:    4603147665287872912 0x8dc528->"MAIN"
                    6343797728819096967 0x8dc4d8->"100200"
string_set[  28]:     101296173680447455 0x8dbd88->"000500"
                    4764204919547260605 0x8dbc78->"OBJECT"
string_set[  29]:    8115490840279316602 0x8dba98->"PROGRAM"
string_set[  32]:   11749562128152811880 0x8dbf18->"000700"
                   14635865553072824514 0x8dbc28->"000400"
string_set[  33]:    3462224256807982264 0x8dc058->"COBOL"
string_set[  35]:   16250758790188753277 0x8dc488->"100100"
string_set[  36]:   18104328237522131839 0x8dbec8->"SECTION"
string_set[  39]:    3000827399044917548 0x8dba28->"DIVISION"
string_set[  40]:    5023835908241093643 0x8dbb38->"ID"
string_set[  42]:    8123926825918656885 0x8dc438->"PROCEDURE"
string_set[  43]:    6530573821985112690 0x8dcc18->"STOP"
                   10110854800518129440 0x8dbfb8->"COMPUTER"
string_set[  48]:   15130070450414024771 0x8dc898->"EOS"
string_set[  50]:    3346600377836954007 0x8dc348->"FILE"
string_set[  52]:    4975450011984783841 0x8dc3e8->"100000"
string_set[  54]:    2856247231245445436 0x8dcd58->"100800"
string_set[  55]:   17675615744022553182 0x8dbe78->"CONFIGURATION"
                   16474229124768219677 0x8dbe28->"000600"
                     159993502864411231 0x8d7318->"000100"
string_set[  56]:    1559822377874549718 0x8dc578->"LOGIC"
                   11683372015007288834 0x8dbcc8->"000900"
string_set[  59]:   14633305175400438363 0x8dc8e8->"100500"
string_set[  60]:   10763892105548688733 0x8dc1e8->"100600"
string_set[  62]:    9459438815593384502 0x8dc2a8->"DATA"
string_set[  64]:    5687493358103633321 0x8dc398->"001200"
string_set[  68]:   11590517852533769637 0x8dc0a8->"000800"
string_set[  70]:     603146580336197870 0x8dc148->"15"
                   12485971477340877982 0x8dbce8->"001000"
string_set[  72]:    2963571779194552878 0x8dc668->"100400"
string_set[  76]:    3031676173105530525 0x8dc7f8->"POSITION"
string_set[  78]:    7989954789217878515 0x8dbdd8->"ENVIRONMENT"
string_set[  79]:   11133740230814649646 0x8dc6b8->"DISPLAY"
string_set[  82]:    6921971865612019692 0x8dc008->"RM"
string_set[  85]:    7372855250431704371 0x8dbae8->"-"
                   10010194838115560012 0x8dc708->"" ""
string_set[  86]:    3506203549115477658 0x8dbb88->"HELLOWORLD"
string_set[  88]:    1632198207615728178 0x8dc758->"LINE"
string_set[  89]:    6637313742931709005 0x8dbd38->"*"
string_set[  90]:     816481200638801389 0x8dbf68->"SOURCE"
string_set[  93]:   13562144813845494497 0x8dc198->"10."
string_set[  94]:    9851758844805121084 0x8dc5c8->"100300"
load_factor = 0.577
bucket_count = 97
max_bucket_size = 3
