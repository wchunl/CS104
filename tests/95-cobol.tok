#  1 "95-cobol.oc"
#  1 "<built-in>"
#  1 "<command-line>"
#  1 "95-cobol.oc"
   4 3.00001 283 TOK_INTCON    000100
   4 3.00008 282 TOK_IDENT     IDENTIFICATION
   4 3.00023 282 TOK_IDENT     DIVISION
   4 4.00001 283 TOK_INTCON    000200
   4 4.00008 282 TOK_IDENT     PROGRAM
   4 4.00015  45 '-'           -
   4 4.00016 282 TOK_IDENT     ID
   4 4.00020 282 TOK_IDENT     HELLOWORLD
   4 5.00001 283 TOK_INTCON    000300
   4 6.00001 283 TOK_INTCON    000400
   4 6.00007  42 '*'           *
   4 7.00001 283 TOK_INTCON    000500
   4 7.00008 282 TOK_IDENT     ENVIRONMENT
   4 7.00020 282 TOK_IDENT     DIVISION
   4 8.00001 283 TOK_INTCON    000600
   4 8.00008 282 TOK_IDENT     CONFIGURATION
   4 8.00022 282 TOK_IDENT     SECTION
   4 9.00001 283 TOK_INTCON    000700
   4 9.00008 282 TOK_IDENT     SOURCE
   4 9.00014  45 '-'           -
   4 9.00015 282 TOK_IDENT     COMPUTER
   4 9.00025 282 TOK_IDENT     RM
   4 9.00027  45 '-'           -
   4 9.00028 282 TOK_IDENT     COBOL
   4 10.00001 283 TOK_INTCON    000800
   4 10.00008 282 TOK_IDENT     OBJECT
   4 10.00014  45 '-'           -
   4 10.00015 282 TOK_IDENT     COMPUTER
   4 10.00025 282 TOK_IDENT     RM
   4 10.00027  45 '-'           -
   4 10.00028 282 TOK_IDENT     COBOL
   4 11.00001 283 TOK_INTCON    000900
   4 12.00001 283 TOK_INTCON    001000
   4 12.00008 282 TOK_IDENT     DATA
   4 12.00013 282 TOK_IDENT     DIVISION
   4 13.00001 283 TOK_INTCON    001100
   4 13.00008 282 TOK_IDENT     FILE
   4 13.00013 282 TOK_IDENT     SECTION
   4 14.00001 283 TOK_INTCON    001200
   4 15.00001 283 TOK_INTCON    100000
   4 15.00008 282 TOK_IDENT     PROCEDURE
   4 15.00018 282 TOK_IDENT     DIVISION
   4 16.00001 283 TOK_INTCON    100100
   4 17.00001 283 TOK_INTCON    100200
   4 17.00008 282 TOK_IDENT     MAIN
   4 17.00012  45 '-'           -
   4 17.00013 282 TOK_IDENT     LOGIC
   4 17.00019 282 TOK_IDENT     SECTION
   4 18.00001 283 TOK_INTCON    100300
   4 18.00008 282 TOK_IDENT     BEGIN
   4 19.00001 283 TOK_INTCON    100400
   4 19.00008 282 TOK_IDENT     DISPLAY
   4 19.00016 285 TOK_STRINGCON " "
   4 19.00020 282 TOK_IDENT     LINE
   4 19.00025 283 TOK_INTCON    1
   4 19.00027 282 TOK_IDENT     POSITION
   4 19.00036 283 TOK_INTCON    1
   4 19.00038 282 TOK_IDENT     ERASE
   4 19.00044 282 TOK_IDENT     EOS
   4 20.00001 283 TOK_INTCON    100500
   4 20.00008 282 TOK_IDENT     DISPLAY
   4 20.00016 285 TOK_STRINGCON "Hello world!"
   4 20.00031 282 TOK_IDENT     LINE
   4 20.00036 283 TOK_INTCON    15
   4 20.00039 282 TOK_IDENT     POSITION
   4 20.00048 283 TOK_INTCON    10.
   4 21.00001 283 TOK_INTCON    100600
   4 21.00008 282 TOK_IDENT     STOP
   4 21.00013 282 TOK_IDENT     RUN
   4 22.00001 283 TOK_INTCON    100700
   4 22.00008 282 TOK_IDENT     MAIN
   4 22.00012  45 '-'           -
   4 22.00013 282 TOK_IDENT     LOGIC
   4 22.00018  45 '-'           -
   4 22.00019 282 TOK_IDENT     EXIT
   4 23.00001 283 TOK_INTCON    100800
   4 23.00008 282 TOK_IDENT     EXIT
