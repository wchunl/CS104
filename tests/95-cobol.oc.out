; 0.  1: # 1 "95-cobol.oc"; 1.  0: 
; 1.  1: # 1 "<built-in>"
; 2.  1: # 1 "<command-line>"
; 3.  1: # 1 "95-cobol.oc"
; 4.  1: 
; 4.  2: 
; 4.  3: 000100 IDENTIFICATION DIVISION.
; 4.  4: 000200 PROGRAM-ID. HELLOWORLD.
; 4.  5: 000300
; 4.  6: 000400*
; 4.  7: 000500 ENVIRONMENT DIVISION.
; 4.  8: 000600 CONFIGURATION SECTION.
; 4.  9: 000700 SOURCE-COMPUTER. RM-COBOL.
; 4. 10: 000800 OBJECT-COMPUTER. RM-COBOL.
; 4. 11: 000900
; 4. 12: 001000 DATA DIVISION.
; 4. 13: 001100 FILE SECTION.
; 4. 14: 001200
; 4. 15: 100000 PROCEDURE DIVISION.
; 4. 16: 100100
; 4. 17: 100200 MAIN-LOGIC SECTION.
; 4. 18: 100300 BEGIN.
; 4. 19: 100400 DISPLAY " " LINE 1 POSITION 1 ERASE EOS.
; 4. 20: 100500 DISPLAY "Hello world!" LINE 15 POSITION 10.
; 4. 21: 100600 STOP RUN.
; 4. 22: 100700 MAIN-LOGIC-EXIT.
; 4. 23: 100800 EXIT.
